README.md

```diff
@@ -1,10 +1,50 @@
 # SQLite on Unikraft
 
-To build and run this application please use the `kraft` script:
+This application starts a SQLite database.
+
+To configure, build and run the application you need to have [kraft](https://github.com/unikraft/kraft) installed.
+
+To be able to interact with the database, configure the application to run on the KVM platform:
+```
+$ kraft configure -p kvm -m x86_64
+```
+
+Build the application:
+```
+$ kraft build
+```
+
+And, finally, run the application:
+```
+$ kraft run
+SQLite version 3.30.1 2019-10-10 20:19:45
+Enter ".help" for usage hints.
+Connected to a transient in-memory database.
+Use ".open FILENAME" to reopen on a persistent database.
+sqlite>
+```
+
+If you want to have more control you can also configure, build and run the application manually.
+
+To configure it for the KVM platform:
+```
+$ make menuconfig
+```
+
+Build the application:
+```
+$ make
+```
+
+Run the application:
+```
+sudo qemu-system-x86_64 -fsdev local,id=myid,path=$(pwd)/fs0,security_model=none \
+                        -device virtio-9p-pci,fsdev=myid,mount_tag=rootfs,disable-modern=on,disable-legacy=off \
```

Comment:
```suggestion
                        -device virtio-9p-pci,fsdev=myid,mount_tag=fs0,disable-modern=on,disable-legacy=off \
```

If we do the change in `kraft.yaml` file for mounting tag to be `fs0` I think we should update also here because most of the people will build with kraft and then use `kraft run` or this command to run the application and we need to have the same mounting tag.